<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagedTool</class>
 <widget class="QMainWindow" name="PackagedTool">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>206</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PackagedTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #000000, stop:0.7 #474e5e);}
QWidget{color: rgb(223, 223, 223);}
QPushButton{border-radius: 3px;border:0px solid transparent;background-color:rgba(88,88,88,150);color:#DDDDDD;height:24;}
QPushButton:hover{background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #f4716e, stop:0.5 transparent);}
QPushButton:pressed{background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #42f875, stop:0.5 transparent);}
QPushButton::disabled{background-color:#7d7d7d;}
QLineEdit{background-color:transparent;}
QComboBox{background-color:transparent;}
QComboBox QAbstractItemView{background-color:transparent;}
QToolButton{border-radius: 3px;border:0px solid transparent;background-color:rgba(88,88,88,150);color:#DDDDDD;height:24;}
QToolButton:hover{background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #f4716e, stop:0.5 transparent);}
QToolButton:pressed{background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #42f875, stop:0.5 transparent);}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_start_package">
      <property name="text">
       <string>开始打包</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_package" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_qt_dir">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>例如： C:\Qt\Qt5.12.12\5.12.12\msvc2017_64\bin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="toolButton_package_dir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>安装程序：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>输出目录</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_output_dir"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_install_dir">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_package_dir"/>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="toolButton_qt_dir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="toolButton_install_dir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="toolButton_output_dir">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>输出文件名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Qt环境路径：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>打包目录：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEdit_output_name"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
